pr-template.md

```diff
@@ -1,20 +1,35 @@
 #### What does this PR do? (please provide any background)
+
+
 #### What tests does this PR have?
+
+
 #### How can this be tested?
+
+
 #### Screenshots / Screencast
+
+
 #### What gif best describes how you feel about this work?
+![](url)
+
 ---
-#### Developer Definition of Done/Quality Checklist (for PR author to complete BEFORE code review):
+
 - [ ] I have checked the [contributing document](../blob/master/CONTRIBUTING.md) and I'm happy for this to be reviewed.
 
-#### Software Engineer or Developer review:
-- [ ] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
-- [ ] I’ve checked for coding anti-patterns.
-- [ ] I’ve checked for appropriate test coverage.
-- [ ] I’ve checked all the tests are passing.
-
-#### Software Engineer or project guru review:
-- [ ] I’ve witnessed the work behaving as expected (this could be on the authors machine or screencast).
-- [ ] I’ve checked for coding anti-patterns.
-- [ ] I’ve checked for appropriate test coverage.
-- [ ] I’ve checked all the tests are passing.
+#### Reviewers
+
+**IP** _(optional)_
+- [ ] :+1:
+
+**Dev**
+- [ ] :+1:
+
+**SE/Guru**
+- [ ] :+1:
+
+By adding a +1 you are confirming you have...
+- Witnessed the work behaving as expected (this could be on the author's machine or screencast).
+- Checked for coding anti-patterns.
+- Checked for appropriate test coverage.
+- Checked all the tests are passing.
```

Comment:
I think in some instances we can merge if one CI has passed and not another - eg Travis passing but not Team City for Tripapp.